README.md

```diff
@@ -90,9 +90,44 @@ and then loaded into the cluster using terraform on cluster build.
 
 Initial set up requires manual steps in the cluster Azure KV.
 
-- create KV certificate CA
-    - The CA API key was created in our digicert service account, and is kept in a secret in the prod TSC Domains KeyVault.
-- generate cert within KV certificate
+#### Add new top domain
+
```

Comment:
Please add this was required for allowing teacherservices.cloud top domain. It is not needed anymore for new clusters under the same top domain.